src/app/containers/MostRead/utilities/index.js

```diff
@@ -0,0 +1,12 @@
+export const mostReadRecordIsFresh = lastRecordTimeStamp => {
+  const now = Date.now();
+  const utcTimestamp = Date.parse(lastRecordTimeStamp);
+  const mostReadLifeSpan = 35 * 60 * 1000; // 35 minutes
```

Comment:
Can this be a constant at the top of the file, for visibility? Same for the 60 days value. 
e.g. `MAXIMUM_STALE_DATA_TIME` for the 35 mins / `SHOULD_RENDER_LAST_UPDATED_TIME` for the 60 days.